core/modules/catalog/components/ProductBundleOption.ts

```diff
@@ -57,13 +60,18 @@ export const ProductBundleOption = {
   methods: {
     setDefaultValues () {
       if (this.option.product_links) {
-        const defaultOption = this.option.product_links.find(pl => { return pl.is_default })
+        let defaultOption
+        if (Array.isArray(this.option.product_links)) {
+          defaultOption = this.option.product_links.find(pl => { return pl.is_default })
+        } else {
+          defaultOption = this.option.product_links
+        }
```

Comment:
```suggestion
      const { product_links } = this.option
      const defaultOption = Array.isArray(product_links) ? product_links.find(pl => pl.is_default) : product_links
```

Reply:
Normally I would write it like this

```typescript
const defaultOption = Array.isArray(product_links) ? product_links.find(pl => {
          return pl.is_default
        }) : product_links
```

as I try to always be under the 120 length in a line. 

What do you think about that?

Reply:
yeah it's ok, but you can break line in the `inline if` just after question mark or colon eg:
```
condition ? // wrap line
 true : // wrap line
 false
```

Reply:
also you don't need brackets here:

```
product_links.find(pl => { return pl.is_default }) -> product_links.find(pl =>pl.is_default)
```

Reply:
I have changed it I can do it 100% like you want it because of eslint :smile:

Reply:
@ResuBaka ESlint always knows better how to make you happier 😆